src/ui/common/src/components/integrations/cards/card.tsx

```diff
@@ -98,6 +94,8 @@ type IntegrationProps = {
 export const IntegrationCard: React.FC<IntegrationProps> = ({
   integration,
 }) => {
+  // TODO: Move the integration's presentatinal component up a level.
```

Comment:
File a Linear task here and we can resolve later, as @kenxu95 is working on the cards at the moment.

Reply:
See ENG-2768